db/migrate/20240724085326_change_error_code_type_on_error_details.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class ChangeErrorCodeTypeOnErrorDetails < ActiveRecord::Migration[7.1]
```

Comment:
Is there any chance we can merge those two migrations? We haven't deployed it anywhere so it is fine to do it in one file

Reply:
💪